setup/apache.yml

```diff
@@ -18,7 +18,7 @@ apache_cosign_deny_friend:     "global=app_cosign_deny_friend default=no"
 
 apache_static_path:
   global: deploy_root
-  output: File.join(deploy_root, app_name, "shared", "public")
+  output: File.join(deploy_root, app_name, "current", "public")
```

Comment:
I feel like this is going to bite us later, but I don't see much of a way around it.
```
current/public/
  assets/ -> shared/public/assets
  system/ -> shared public/system
  uploads/ -> shared/public/uploads
  blog/
  about/
shared/public/
  assets/
  system/
  uploads/
```
It seems like the jekyl generated pages are just assets.  I'm wondering if heliotrope shouldn't have just made `public/` a shared directory where `current/public/ -> shared/public`.  That would mean they'd have to refactor the deploy script so that it wouldn't be trying to link anything under `public/` because those would end up being circular at that point.